init.lua

```diff
@@ -383,6 +425,9 @@ function RECV_MSG.ReceivedItems(msg)
 	end
 
 	local orb_count = 0
+	if not Cache.ItemDelivery:is_set(1) and not GameHasFlagRun("ap_spawn_kill_saver") then
```

Comment:
```suggestion
	if Cache.ItemDelivery:num_items() == 0 and not GameHasFlagRun("ap_spawn_kill_saver") then
```
probably more easily understood